src/data/battle_moves.h

```diff
@@ -2724,7 +2724,7 @@ const struct BattleMove gBattleMoves[MOVES_COUNT] =
         .secondaryEffectChance = 10,
         .target = MOVE_TARGET_SELECTED,
         .priority = 0,
-        .flags = FLAG_MAKES_CONTACT | FLAG_PROTECT_AFFECTED | FLAG_MIRROR_MOVE_AFFECTED | FLAG_SHEER_FORCE_BOOST,
```

Comment:
Is EFFECT_THAW_HIT needed anymore, now that the flag takes its place? Personally, I'd get rid of that flag and put a placeholder there for a new move effect.

Reply:
Good point. EFFECT_THAW_HIT is only used for Flame Wheel and Sacred Fire, so it could be replaced with EFFECT_BURN_HIT in both cases.

Reply:
Also done. Can revert the last commit if we want to keep EFFECT_THAW_HIT defined.